src/geode/geosciences_io/mesh/gocad_common.cpp

```diff
@@ -38,6 +38,49 @@ namespace
     static constexpr char EOL{ '\n' };
     static constexpr char SPACE{ ' ' };
 
+    std::vector< std::string > split_composed_strings(
+        absl::string_view string )
+    {
+        std::vector< std::string > merged;
+        const auto tokens = geode::string_split( string );
+        geode::index_t token_id{ 0 };
+        while( token_id < tokens.size() )
+        {
+            std::string token;
+            if( tokens[token_id] == "\"" )
+            {
+                token_id++;
+                OPENGEODE_ASSERT( token_id < tokens.size() );
+                while( tokens[token_id].back() != '\"'
+                       && token_id < tokens.size() )
+                {
+                    absl::StrAppend( &token, " ", tokens[token_id] );
+                    token_id++;
+                    OPENGEODE_ASSERT( token_id < tokens.size() );
+                }
+                absl::StrAppend( &token, " ", tokens[token_id] );
+                token.erase( token.size() - 1, 1 );
```

Comment:
Is it to remove the " character ? If yes, I'm not sure it is also removed from the first added token

Reply:
Yes but I do not care. Th first token cannot end but a ".